IBX-8590: Reorganizing documentation in terms of Product Guides (#2479)
ibexa/documentation-developer: docs/customer_management/customer_portal_guide.md

### Simplified shopping process

Business account helps streamline the B2B shopping process with all the paperwork, payment, 
and other administrative tasks converted into a few simple steps with prefilled forms, billing addresses, shipping addresses and more. 
Making your site a go-to place for company orders.

### Better customer experience

In the era of internet, customers expect quick, accessible and excellent quality service and user experience from every business they associate with. 
Customer portals offer a seamless self-service experience by providing complete 24/7 access to relevant, up-to-date information and customer support.

### Client encouragement

Price strategies are a great way to build and maintain strong relationships with your trading partners. 
With special prices available to B2B clients, you can offer the best deals in highly competitive markets. 
Those discounts may be a great encouragement to convince big buyers to choose your business over other options. 
Competitive prices impact not only the size of the customer base, 
they affect every customer’s purchasing strategy, including the diversity, frequency, and volume of their orders.

### Cost benefits

> Customer portals help you to automate tasks that otherwise would be done by your employees manually, such as customer services, checking shipment status. 

Reviewer: ```suggestion
Customer portals help yo automate tasks that otherwise would be done by your employees manually, such as customer services, or checking shipment status. 
```

> An additional benefit of customer portals is their availability 24/7. 
> Thus, reducing the need to allocate resources to extend working hours or hire more employees.

Reviewer: ```suggestion
An additional benefit of customer portals is their availability 24/7, reducing the need to allocate resources to extend working hours or hire more employees.
```
